tests/test_apiclient.py

```diff
@@ -0,0 +1,72 @@
+import mock
```

Comment:
Do we use the three import blocks in this project? (stdlib, external libs, project libs)